modules/ROOT/pages/design-create-publish-api-specs.adoc

```diff
@@ -5,11 +5,15 @@ endif::[]
 :imagesdir: ../assets/images
 :page-aliases: general:getting-started:design-an-api.adoc, design-migrating-ms.adoc
 
-API Designer provides a web-based interface for designing, documenting, testing, and sharing API specifications and fragments.
+API Designer provides a web-based interface for designing API specifications and fragments in:
+
+* RESTful API Modeling Language (RAML) version 0.8 or 1.0
+* OpenAPI specification (OAS) version 2.0 or 3.0
+* AsyncAPI 2.0
 
 == Designing
```

Comment:
Designing _what_?
And _only_ designing?
Historically, we have not used gerunds in our titles; I am hoping we can change that for non-task topics